packages/lit-dev-content/src/code-language-preference.ts

```diff
@@ -30,10 +56,26 @@ export const getCodeLanguagePreference = (): CodeLanguagePreference =>
  * a change event on window.
```

Comment:
May be worth adding some docs that this function now has two behaviors: what is written, and initiating a user prompted language change.

Reply:
Good idea, done!